core/provisioning-api/src/test/java/org/apache/syncope/core/provisioning/api/IntAttrNameParserTest.java

```diff
@@ -362,4 +363,13 @@ public void invalid() {
             assertNotNull(e);
         }
     }
+
+    @Test
+    public void issue1894() throws ParseException {
```

Comment:
This method should be named `issueSYNCOPE1894()` to follow the current convention